.github/workflows/test_ci.yml

```diff
@@ -0,0 +1,91 @@
+name: Test
+
+on:
+  push:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        include:
+          - name: ubuntu_16_04-x86_64
+            os: ubuntu-16.04
+            TARGET_CPU: x86-64
+
+          - name: ubuntu_18_04-x86_64
+            os: ubuntu-18.04
+            TARGET_CPU: x86-64
+
+          - name: ubuntu_20_04-x86_64
+            os: ubuntu-20.04
+            TARGET_CPU: x86-64
+
+          - name: osx_11_0-x86_64
+            os: macos-11.0
+            TARGET_CPU: x86-64
+
+          - name: osx_10_11-x86_64
+            os: macos-10.15
+            TARGET_CPU: x86-64
+
+          - name: win-x86_64
```

Comment:
We need to test windows 32-bit. We should be able to run tests that are cross-compiled for 32-bit on 64-bit, since 32-bit apps run fine on windows 64.